tests/test_benchmark.py

```diff
@@ -41,6 +42,22 @@ class BenchmarkConfig:
     num_batches: int
     steps_per_batch: int
     verbose: bool
+    generate_plots: bool
+
+
+def plot_batch_times(steps_per_batch: int, dt: float, batch_times: List[float], label: str):
```

Comment:
nit: would be useful to indicate the expected units of `dt` and `batch_times` here (getting a bit confused trying to follow the conversion below).

Reply:
Added in https://github.com/proteneer/timemachine/pull/1375/commits/aac5753f56de435af88715ad77c6af07542100e5